include/string.h

```diff
@@ -22,11 +22,33 @@ extern "C" {
 
 #include <bits/alltypes.h>
 
-void *memcpy (void *__restrict, const void *__restrict, size_t);
-void *memmove (void *, const void *, size_t);
-void *memset (void *, int, size_t);
-int memcmp (const void *, const void *, size_t);
-void *memchr (const void *, int, size_t);
+//returns a pointer to dest : Copies n bytes form src to the memory region dest points to 
```

Comment:
Please capitalize the first letter of the comment. Also add a space between `//` and `Returns`.

`// Returns a pointer to dest. Copies ...`